<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-07-07T16:57:23 CEST" name="Default Suite" finished-at="2021-07-07T16:57:44 CEST" duration-ms="21590">
    <groups>
    </groups>
    <test started-at="2021-07-07T16:57:23 CEST" name="HRProject" finished-at="2021-07-07T16:57:44 CEST" duration-ms="21590">
      <class name="PersonalInfoTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:PersonalInfoTest@2f67a4d3]" started-at="2021-07-07T16:57:23 CEST" name="beforeTest" finished-at="2021-07-07T16:57:29 CEST" duration-ms="6708" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="LogIn()[pri:1, instance:PersonalInfoTest@2f67a4d3]" started-at="2021-07-07T16:57:29 CEST" name="LogIn" description="User is logging in" finished-at="2021-07-07T16:57:35 CEST" duration-ms="5703" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
The title is incorrect expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
The title is incorrect expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at PersonalInfoTest.LogIn(PersonalInfoTest.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogIn -->
        <test-method signature="PersonalInfo()[pri:2, instance:PersonalInfoTest@2f67a4d3]" started-at="2021-07-07T16:57:35 CEST" name="PersonalInfo" description="User is editing Personal Information" finished-at="2021-07-07T16:57:42 CEST" duration-ms="6981" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PersonalInfo -->
        <test-method signature="LogOut()[pri:3, instance:PersonalInfoTest@2f67a4d3]" started-at="2021-07-07T16:57:42 CEST" name="LogOut" description="User is logging out" finished-at="2021-07-07T16:57:43 CEST" duration-ms="1001" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogOut -->
        <test-method is-config="true" signature="close()[pri:0, instance:PersonalInfoTest@2f67a4d3]" started-at="2021-07-07T16:57:43 CEST" name="close" finished-at="2021-07-07T16:57:44 CEST" duration-ms="1024" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- PersonalInfoTest -->
    </test> <!-- HRProject -->
  </suite> <!-- Default Suite -->
</testng-results>
